fastlane_core/lib/fastlane_core/tag_version.rb

```diff
@@ -6,7 +6,14 @@ module FastlaneCore
   class TagVersion < Gem::Version
     class << self
       def correct?(tag)
-        superclass.correct?(version_number_from_tag(tag))
+        result = superclass.correct?(version_number_from_tag(tag))
+
+        # It seems like depending on the Ruby env, the result is
```

Comment:
NOTE: The behavior of `correct?` has been changed at this point.
https://github.com/rubygems/rubygems/commit/4a1180f0290ecc49b518107b14aff872c475dced

Reply:
Yah, so now we support both the old and the new version, right? :)

Reply:
This looks like a good improvement, but given the way it is used in `import_from_git`, does this actually change anything? It would still select any tag that returned falsey from `correct?`. And I think there are plenty of methods in Ruby that are basically Boolean but may also return nil.